gradle/libs.versions.toml

```diff
@@ -4,7 +4,8 @@ agpAlpha = "8.6.1"
 anvil = "2.5.0-beta11"
 bugsnagGradle = "8.1.0"
 circuit = "0.23.1"
-compose-jb = "1.6.11"
+compose-jb = "1.7.0-alpha03"
```

Comment:
Let's use the latest beta - https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.7.0-beta02

Reply:
aa32432f1ca9f371e9c8ebfe96d96a369f54c217